src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -318,7 +318,7 @@ public void update(UpdateOptions updateOptions) {
         updateOptions.commentOption.ifPresent(s -> comments = s);
         updateOptions.googleIdOption.ifPresent(s -> googleId = s);
         updateOptions.teamNameOption.ifPresent(s -> team = s);
-        updateOptions.sectionNameOption.ifPresent(s -> section = s);
+        updateOptions.sectionNameOption.ifPresent(s -> section = s.isEmpty() ? Const.DEFAULT_SECTION : s);
```

Comment:
Here you only change the logic of updating student. The problem may still exist when you enroll a new student. Although previously I said to modify the DB layer (but note that this `StudentAttributes` actually is not DB layer), to isolate this issue, it is better to fix this problem in API layer. i.e. change the `studentEnrollRequest` to settle default section name if empty

Reply:
Although it's not apparent, the change does cover both Enroll and Update actions because the `update()` method is used by `build()` which is used by both actions. 

But if it's better to do it in the API layer should I remove this from the `StudentAttributes`? or have both

Reply:
Yes, it's better to remove the `StudentAttributes` change, and test it manually after you finish the API change.

Reply:
I couldn't find a good way to place it in`StudentsEnrollRequest` so instead placed it in `EnrollStudentsAction` and `UpdateStudentAction`

Reply:
A good way is to place it in `getter method` of `StudentEnrollRequest`. In addition, you may also want to add comment about this change, and make it covered by test

Reply:
Updated. Moved the changes to `StudentEnrollRequest` and `StudentUpdateRequest`